src/widgets/capture/capturewidget.cpp

```diff
@@ -322,7 +322,8 @@ void CaptureWidget::paintEvent(QPaintEvent*)
     painter.setClipRect(rect());
 
     if (m_showInitialMsg) {
-#if (defined(Q_OS_MAC) || defined(Q_OS_MAC64) || defined(Q_OS_MACOS) ||        \
```

Comment:
Do you happen to know if this will also work on Windows so we can remove the platform check? I don't have a Windows box handy.

Reply:
I don't have Windows but I do have a Windows 10 VM set up. After my class ends I will check it out and see if it works.

Reply:
I just tested the automatic build from 6523c34 on Windows 10. Seems like it works, but there is one problem - the tool settings window pops up on the primary monitor, the rest seems to open on the current active monitor like intended.

I am not sure on why this is happening and I have no experience on working with Qt applications.